tests/components/ct-angular/tests/hooks.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/experimental-ct-angular';
+import type { HooksConfig } from 'playwright';
+import { InjectComponent } from '@/components/inject.component';
+import { AppComponent } from '@/app.component';
+
+
+test('navigate to a page by clicking a link', async ({ page, mount }) => {
```

Comment:
Actually, the plugin doesn't provide any routing support, yet, the test is actually only testing that the hooks work and it is using the routing as a hook side effect. It could (should?) have been something else.

That said we can use another way of testing hooks and separate a routing teat just as means of documentation.